src/rules/importBlacklistRule.ts

```diff
@@ -76,7 +76,7 @@ class NoRequireFullLibraryWalker extends Lint.RuleWalker {
     }
 
     public visitImportEqualsDeclaration(node: ts.ImportEqualsDeclaration) {
-        let moduleReference = <ts.ExternalModuleReference> node.moduleReference;
+        const moduleReference = <ts.ExternalModuleReference> node.moduleReference;
```

Comment:
you mean the angle bracket assertion? this is an outdated diff; I did that in the other PR